### ISSUE TITLE
How to implement swipe to delete in SortedListAdapter

### ISSUE THREAD

**Issue by yaturner** [Wrdlbrnft/SortedListAdapter#3]
I am a big fan of your sortedListAdapter, I have implemented search and databinding and I am very pleased with the results. I would now like to implement swipe to delete, but I am unsure of how to do it, can you help me?

**Comment by Wrdlbrnft** [comment]
You implement it just like with any other `Adapter`. The `RecyclerView` comes with a helper class to simplify implementing swipe to dismiss and reorder patterns, it's called `ItemTouchHelper`. You can find more information about it [**here**](https://developer.android.com/reference/android/support/v7/widget/helper/ItemTouchHelper.html). If you want a more in depth guide just google around a bit.

**Comment by yaturner** [comment]
thank you for your help, I am a little lost though, the example I found
overrides onBindViewHolder to check for pending delete. I tried overriding
the method in my adapter which extends SortedListAdapter, but i get an
error from Java about the override having the same erasure as the method in
SortedListAdapter. Do I need to implement it in your code? Also the code
assumes that I can access the List<item> for the ListView in the adapter,
how do I do that. Sorry to sound so stupid but I am using RecyclerView for
the first time.
